framework/include/transfers/MultiAppTransfer.h

```diff
@@ -195,7 +195,8 @@ class MultiAppTransfer : public Transfer
    * see StochasticToolsTransfer for an example.
    */
   void checkMultiAppExecuteOn();
-
+  void checkPostProcessorExecuteOn(std::string to_check);
+  void checkAgainstPPExec(std::string to_check, FEProblemBase & problem);
```

Comment:
```suggestion
  void checkPostprocessorExecuteOn(std::string to_check, FEProblemBase & problem);
```

Reply:
neither PP or Exec are that common in the code

Reply:
I named it this to abstract out the repeated code from the parent function. The actual function being called by the Transfers is already called checkPostProcessorExecuteOn so this suggestion does not work as it's not an overloaded version of the same function.

Reply:
seems like we need another name still
funny that I improvised the same name as the other routine